packages/suite/src/types/coinmarket/coinmarketOffers.ts

```diff
@@ -108,8 +80,8 @@ export type CoinmarketOffersContextValues<T extends CoinmarketTradeType> =
 export interface CoinmarketCryptoAmountProps {
     amountInCrypto?: boolean | undefined;
     sendAmount: string | number | undefined;
-    sendCurrency: string | undefined;
+    sendCurrency: CryptoId | string | undefined; // TODO: ???
```

Comment:
If `CryptoId` is "more strict" string then we should not allow `string` here (or am I mistaken?)
```suggestion
    sendCurrency: CryptoId | undefined;
```
And what should we do with the TODO?